features/prompt-registry.mdx

```diff
@@ -487,7 +487,38 @@ The following code snippet shows you how to do this:
 
 <CodeGroup>
 
+```python Python SDK (Metadata)
+template = promptlayer.templates.get(
+  "my_prompt_with_functions",
+  {
+    "provider": "openai",
+    "input_variables": {
+      "location": "Boston, MA",
+      "unit": "fahrenheit",
+    }
+  }
+)
+response = openai.chat.completions.create(**template["llm_kwargs"])
+```
+
 ```python Python SDK
+template = promptlayer.templates.get(
+  "my_prompt_with_functions",
+  {
+    "provider": "openai",
+    "input_variables": {
+      "location": "Boston, MA",
+      "unit": "fahrenheit",
+    }
+  }
+)
+response = openai.chat.completions.create(
```

Comment:
Why is this `openai.chat` instead of `client.chat`?

Reply:
updated to client